README.md

```diff
@@ -30,6 +30,9 @@ OR
 Respects `editor.formatOnSave` setting.
 
 ### Settings
+#### prettier.eslintIntegration (default: true)
```

Comment:
Isn't the default `false` here? https://github.com/esbenp/prettier-vscode/pull/92/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R40